features/mbedtls/platform/inc/platform_mbed.h

```diff
@@ -17,6 +17,12 @@
  *  This file is part of mbed TLS (https://tls.mbed.org)
  */
 
+
+#ifndef __PLATFORM_MBED__H__
+#define __PLATFORM_MBED__H__
+
+#include "default_random_seed.h"
```

Comment:
Why do we need to include this header? 

There should be no "default random seed". That sounds really dangerous. (And not very random).

Reply:
This header contains the default read/write functions that supposed to be registered with the Mbed TLS `NV_SEED` feature.